src/ast/passes/resource_analyser.cpp

```diff
@@ -222,6 +236,11 @@ void ResourceAnalyser::prepare_mapped_printf_ids()
   }
 }
 
+bool ResourceAnalyser::uses_usym_table(const std::string &fun)
+{
+  return fun == "usym" || fun == "func" || fun == "ustack";
```

Comment:
ustack is listed here, but it doesn't use the usym table, right?

Reply:
`ustack` uses `resolve_usym` to resolve addresses in the stack to symbols, as shown in your comment above, or did you mean something else?